Formula/mfoc.rb

```diff
@@ -19,11 +20,14 @@ class Mfoc < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "42a1e94b179175e30631ae1d85e59a8d106def94007da48b2c98ccf09e16b13f"
   end
 
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
   depends_on "pkg-config" => :build
   depends_on "libnfc"
   depends_on "libusb"
 
   def install
+    system "autoreconf", "-is"
```

Comment:
Why?

Reply:
As described in readme https://github.com/nfc-tools/mfoc#build-from-source